cem/support-typedef-jsdoc-utils.js

```diff
@@ -44,27 +44,43 @@ export function getTypesFromConstructor (constructorNode, ts, retrievePrivate =
 
 }
 
+const BUILTIN_TYPES = ['String', 'Boolean', 'Number', 'BigInt', 'Date', 'Map', 'Set', 'WeakMap', 'WeakSet', 'Object'];
```

Comment:
Should we add `Promise` and maybe `Symbol`, not sure. I'll let you decide :wink: